### PR TITLE
docs: mark as archived

### PR DESCRIPTION
**Pull request by nickcernis**
- Marks repo as archived ahead of archiving this repo. (The Local team now develops the components library in the same private repo containing the core Local code. We will continue to publish local-components updates to npm from there.)
- Updates link to Playground.

Preview new Readme here:

https://github.com/getflywheel/local-components/tree/nc/archive-notice#readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
+Local Components is the official component library for [Local](https://localwp.com/).
 
 ## What's included in this component library?
 
-We appreciate curious minds and that's a great question!
-Take a quick look at our living component documentation [here](https://getflywheel.github.io/local-components/).
+Take a quick look at our [living component documentation](https://master--62d18352969d7c650b935483.chromatic.com/).
 
-In additional to a quickly growing set of React components, we also have SVGs, SASS partials, and more to come!
+In additional to a quickly growing set of React components, we also have SVGs, SASS partials, and more to come.
```
Comment (kienstra) [inline README.md:9]:
```suggestion
In addition to a quickly growing set of React components, we also have SVGs, SASS partials, and more to come.
```

Comment (kienstra) [reply README.md:9]: Unrelated, but something I noticed

**Review by kienstra — DISMISSED** [review]
Hi @nickcernis,
This looks good!

**Review by dreamwhisper — DISMISSED** [review]

**Review by bgturner — DISMISSED** [review]
Looks good!

File: README.md (hunk 1 above)
Comment (nickcernis) [reply README.md]: Thanks, @kienstra, ready for re-review!

**Comment by nickcernis** [comment]
Applied suggested change from https://github.com/getflywheel/local-components/pull/334#discussion_r1422649146, ready for re-review.

**Review by bgturner — APPROVED** [review]
🔥

**Pull request merged**